### PR TITLE
Add helper funtions for migrating the wallet-proxy.

### PR DESCRIPTION
**Pull request by vikt0r0**
## Purpose

Add helpers that are useful for migrating the wallet-proxy to the node GRPC API v2.

## Changes

Added a few helper functions.

## Checklist

- [X] My code follows the style of this project.
- [X] The code compiles without warnings.
- [X] I have performed a self-review of the changes.
- [X] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [X] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: src/Concordium/Client/Runner/Helper.hs [hunk 1]
```diff
@@ -149,3 +156,29 @@ getResponseValueOrDie :: (MonadIO m)
...
+-- |Get the 'blockhash' header value of a @CIHeaderList@ if present.
+-- Fails with an error message if the header was not present in the
+-- list of headers or if the header value could not be @read@ into a
+-- @BlockHash@. Returns a @BlockHash@ @read@ from the header value
+-- otherwise.
+getBlockHashHeader :: (MonadFail m) => CIHeaderList -> m Types.BlockHash
+getBlockHashHeader hs =
+  case List.find (("blockhash"==) . fst) hs of
+    Just hd ->
+      case readEither . Text.unpack . decodeLatin1 $ snd hd of
```
Comment (abizjak) [inline src/Concordium/Client/Runner/Helper.hs:181]: Why are we using decodeLatin1 here?

**Review by Bargsteen — APPROVED** [review]
LGTM if you add the `'` and handle it ;)

File: src/Concordium/Client/Runner/Helper.hs [hunk 2]
```diff
@@ -149,3 +156,29 @@ getResponseValueOrDie :: (MonadIO m)
...
+-- |Get the 'blockhash' header value of a @CIHeaderList@ if present.
+-- Fails with an error message if the header was not present in the
+-- list of headers or if the header value could not be @read@ into a
+-- @BlockHash@. Returns a @BlockHash@ @read@ from the header value
+-- otherwise.
+getBlockHashHeader :: (MonadFail m) => CIHeaderList -> m Types.BlockHash
+getBlockHashHeader hs =
+  case List.find (("blockhash"==) . fst) hs of
+    Just hd ->
+      case readEither . Text.unpack . decodeUtf8 $ snd hd of
```
[the remark this answers is not in the document]
Comment (vikt0r0) [reply src/Concordium/Client/Runner/Helper.hs]: Absolutely! I fixed it.

**Review by abizjak — APPROVED** [review]

**Comment by abizjak** [comment]
@vikt0r0 why is this still not merged?

**Pull request merged**